include/freerdp/settings.h

```diff
@@ -1127,6 +1127,7 @@ struct rdp_settings
 	ALIGN64 BOOL   AsyncChannels;           /* 1546 */
 	ALIGN64 BOOL   AsyncTransport;          /* 1547 */
 	ALIGN64 BOOL   ToggleFullscreen;        /* 1548 */
+	ALIGN64 BOOL   Floatbar;        
```

Comment:
The numbers in the comments are there for a reason, to maintain ABI compatibility replace one of the reserved spots, do not just insert new variables